components/icon/index.en-US.md

```diff
@@ -0,0 +1,125 @@
+---
+category: Components
+chinese: 图标
+type: Basic
+english: Icon
+toc: false
+---
+
+Meaningful vector graphics, each one can defeat an enemy.
+
+## The specification of naming icons
+
+We provide semantic name for every icon, and naming rules are as follows: 
+
+- Scanning line icon has the similar name with its solid one，but it's distinguished by `-o`, for example, `question-circle`(solid) and `question-circle-o`(scanning line);
+
+- Naming sequence：`[icon's name]-[shape, optional]-[Scanning line or not]-[direction, optional]`.
+
+## How to use
+
+Use tag `<Icon />` to define an icon component, and appoint its type, for example:
```

Comment:
Use tag `<Icon />` to create an icon and set its type in the `type` prop, for example: